# Copyright (C) 2022 Aleksandr Migunov

# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 3 of the License, or
# (at your option) any later version.

# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
  
# You should have received a copy of the GNU General Public License
# along with this program. If not, see <https://www.gnu.org/licenses/>.

# -*- coding: utf-8 -*-

from PySide2 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(800, 600)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(10, 10, 241, 21))
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(400, 10, 201, 21))
        self.label_2.setObjectName("label_2")
        self.textEdit = QtWidgets.QTextEdit(self.centralwidget)
        self.textEdit.setGeometry(QtCore.QRect(10, 40, 385, 500))
        self.textEdit.setObjectName("textEdit")
        self.textEdit_2 = QtWidgets.QTextEdit(self.centralwidget)
        self.textEdit_2.setGeometry(QtCore.QRect(405, 40, 385, 500))
        self.textEdit_2.setObjectName("textEdit_2")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 27))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtWidgets.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuNew = QtWidgets.QMenu(self.menuFile)
        self.menuNew.setObjectName("menuNew")
        self.menuOpen = QtWidgets.QMenu(self.menuFile)
        self.menuOpen.setObjectName("menuOpen")
        self.menuSave = QtWidgets.QMenu(self.menuFile)
        self.menuSave.setObjectName("menuSave")
        self.menuConvert = QtWidgets.QMenu(self.menubar)
        self.menuConvert.setObjectName("menuConvert")
        self.menuTo_Ingush = QtWidgets.QMenu(self.menuConvert)
        self.menuTo_Ingush.setObjectName("menuTo_Ingush")
        self.menuTo_Avar = QtWidgets.QMenu(self.menuConvert)
        self.menuTo_Avar.setObjectName("menuTo_Avar")
        self.menuTo_Chechen = QtWidgets.QMenu(self.menuConvert)
        self.menuTo_Chechen.setObjectName("menuTo_Chechen")
        self.menuHelp = QtWidgets.QMenu(self.menubar)
        self.menuHelp.setObjectName("menuHelp")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionExit = QtWidgets.QAction(MainWindow)
        self.actionExit.setObjectName("actionExit")
        self.actionNew_original = QtWidgets.QAction(MainWindow)
        self.actionNew_original.setObjectName("actionNew_original")
        self.actionNew_converted = QtWidgets.QAction(MainWindow)
        self.actionNew_converted.setObjectName("actionNew_converted")
        self.actionOpen_original = QtWidgets.QAction(MainWindow)
        self.actionOpen_original.setObjectName("actionOpen_original")
        self.actionOpen_converted = QtWidgets.QAction(MainWindow)
        self.actionOpen_converted.setObjectName("actionOpen_converted")
        self.actionSave_original = QtWidgets.QAction(MainWindow)
        self.actionSave_original.setObjectName("actionSave_original")
        self.actionSave_converted = QtWidgets.QAction(MainWindow)
        self.actionSave_converted.setObjectName("actionSave_converted")
        self.actionIngush_with_original = QtWidgets.QAction(MainWindow)
        self.actionIngush_with_original.setObjectName("actionIngush_with_original")
        self.actionIngush_without_original = QtWidgets.QAction(MainWindow)
        self.actionIngush_without_original.setObjectName("actionIngush_without_original")
        self.actionAvar_with_original = QtWidgets.QAction(MainWindow)
        self.actionAvar_with_original.setObjectName("actionAvar_with_original")
        self.actionAvar_without_original = QtWidgets.QAction(MainWindow)
        self.actionAvar_without_original.setObjectName("actionAvar_without_original")
        self.actionChechen_with_original = QtWidgets.QAction(MainWindow)
        self.actionChechen_with_original.setObjectName("actionChechen_with_original")
        self.actionChechen_without_original = QtWidgets.QAction(MainWindow)
        self.actionChechen_without_original.setObjectName("actionChechen_without_original")
        self.actionAbout = QtWidgets.QAction(MainWindow)
        self.actionAbout.setObjectName("actionAbout")
        self.actionAbout_Qt = QtWidgets.QAction(MainWindow)
        self.actionAbout_Qt.setObjectName("actionAbout_Qt")
        self.menuNew.addAction(self.actionNew_original)
        self.menuNew.addAction(self.actionNew_converted)
        self.menuOpen.addAction(self.actionOpen_original)
        self.menuOpen.addAction(self.actionOpen_converted)
        self.menuSave.addAction(self.actionSave_original)
        self.menuSave.addAction(self.actionSave_converted)
        self.menuFile.addAction(self.menuNew.menuAction())
        self.menuFile.addAction(self.menuOpen.menuAction())
        self.menuFile.addAction(self.menuSave.menuAction())
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionExit)
        self.menuTo_Ingush.addAction(self.actionIngush_with_original)
        self.menuTo_Ingush.addAction(self.actionIngush_without_original)
        self.menuTo_Avar.addAction(self.actionAvar_with_original)
        self.menuTo_Avar.addAction(self.actionAvar_without_original)
        self.menuTo_Chechen.addAction(self.actionChechen_with_original)
        self.menuTo_Chechen.addAction(self.actionChechen_without_original)
        self.menuConvert.addAction(self.menuTo_Ingush.menuAction())
        self.menuConvert.addAction(self.menuTo_Avar.menuAction())
        self.menuConvert.addAction(self.menuTo_Chechen.menuAction())
        self.menuHelp.addAction(self.actionAbout)
        self.menuHelp.addAction(self.actionAbout_Qt)
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuConvert.menuAction())
        self.menubar.addAction(self.menuHelp.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Converter in Python with PyQt"))
        self.label.setText(_translate("MainWindow", "<html><head/><body><p><span style=\" font-weight:600;\">Original list</span></p></body></html>"))
        self.label_2.setText(_translate("MainWindow", "<html><head/><body><p><span style=\" font-weight:600;\">Converted list</span></p></body></html>"))
        self.menuFile.setTitle(_translate("MainWindow", "File"))
        self.menuNew.setTitle(_translate("MainWindow", "New"))
        self.menuOpen.setTitle(_translate("MainWindow", "Open"))
        self.menuSave.setTitle(_translate("MainWindow", "Save"))
        self.menuConvert.setTitle(_translate("MainWindow", "Convert"))
        self.menuTo_Ingush.setTitle(_translate("MainWindow", "To Ingush"))
        self.menuTo_Avar.setTitle(_translate("MainWindow", "To Avar"))
        self.menuTo_Chechen.setTitle(_translate("MainWindow", "To Chechen"))
        self.menuHelp.setTitle(_translate("MainWindow", "Help"))
        self.actionExit.setText(_translate("MainWindow", "Exit"))
        self.actionNew_original.setText(_translate("MainWindow", "New original"))
        self.actionNew_converted.setText(_translate("MainWindow", "New converted"))
        self.actionOpen_original.setText(_translate("MainWindow", "Open original"))
        self.actionOpen_converted.setText(_translate("MainWindow", "Open converted"))
        self.actionSave_original.setText(_translate("MainWindow", "Save original"))
        self.actionSave_converted.setText(_translate("MainWindow", "Save converted"))
        self.actionIngush_with_original.setText(_translate("MainWindow", "Ingush with original"))
        self.actionIngush_without_original.setText(_translate("MainWindow", "Ingush without original"))
        self.actionAvar_with_original.setText(_translate("MainWindow", "Avar with original"))
        self.actionAvar_without_original.setText(_translate("MainWindow", "Avar without original"))
        self.actionChechen_with_original.setText(_translate("MainWindow", "Chechen with original"))
        self.actionChechen_without_original.setText(_translate("MainWindow", "Chechen without original"))
        self.actionAbout.setText(_translate("MainWindow", "About"))
        self.actionAbout_Qt.setText(_translate("MainWindow", "About Qt"))
